cmd/kubeadm/app/cmd/init.go

```diff
@@ -145,8 +156,8 @@ func NewInit(cfgPath string, cfg *kubeadmapi.MasterConfiguration, skipPreFlight
 	}
 
 	if !skipPreFlight {
-		fmt.Println("Running pre-flight checks")
-		err := preflight.RunInitMasterChecks()
+		fmt.Println("<cmd/init> Running pre-flight checks")
```

Comment:
I've removed it, so that the scope of this PR makes more sense, I do think it still has to be added.